docs/pages/distribution/runtime-versions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Runtime Versions
+---
+
+> Custom runtime versions are not supported on the classic build system (`expo build`); these apps will always use the SDK version as the basis for determining runtime compatibility. 
+
+Over-the-air updates with `expo-updates` work only in apps with a [compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. By default, the runtime version is the Expo SDK version: the runtime is the set of libraries in the Expo SDK. If you choose to customize your runtime by adding or removing modules with native code, the Expo SDK version will no longer properly describe the runtime of the app. In these cases, you need to specify a `runtimeVersion` to ensure your updates are delivered only to compatible clients.  This `runtimeVersion` should be updated whenever you update your project's native modules and change the JS–native interface.
+
+A runtime version must conform to this [description](../versions/latest/config/app/#runtimeversion)
```

Comment:
```suggestion
The runtime version string must conform to [this format](/versions/latest/config/app.md#runtimeversion).
```

can we inline the human regex description here? it seems worthwhile to avoid having to jump to another page. we could possibly make a react component that allows us to refer to the runtimeVersion in the schema to print it here so that we don't have to just copy and paste it

Reply:
I like the inlining, making an issue out of it.